### PR TITLE
Fix web vs native packaging for libparsec in client

### PR DESCRIPTION
**Pull request by touilleMan**
# What has changed ?

- [ ] Does this PR affect the User ?

### CODE REVIEW

File: oxidation/client/package.json [hunk 1]
```diff
@@ -3,28 +3,22 @@
     "version": "0.0.1",
     "private": true,
     "scripts": {
-        "serve": "vue-cli-service serve",
         "postinstall": "cd electron && npm install",
         "test:unit": "vue-cli-service test:unit",
         "test:e2e": "vue-cli-service test:e2e",
         "lint": "vue-cli-service lint",
-        "build": "npx ionic build",
```
Comment (NicoTuxx) [inline oxidation/client/package.json:11]: ionic build was the same than 'vue-cli service build' no ?

File: oxidation/client/package.json [hunk 2]
```diff
@@ -3,28 +3,22 @@
...
         "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
         "electron:release": "npm run electron:copy:release && cd electron && npm run electron:release",
-        "web:libparsec:build": "cd ../bindings/web && wasm-pack build --dev --target web",
-        "web:libparsec:build:release": "cd ../bindings/web && wasm-pack build --release --target web",
-        "web:libparsec:copy": "cp -r ../bindings/web/pkg . || xcopy ..\\bindings\\web\\pkg pkg /e /s /y /i",
-        "web:copy": "npm run web:libparsec:build && npm run web:libparsec:copy",
-        "web:copy:release": "npm run web:libparsec:build:release && npm run web:libparsec:copy",
-        "web:open": "npm run web:copy && vue-cli-service serve",
+        "web:serve": "vue-cli-service serve",
         "web:release": "npm run web:copy:release && vue-cli-service build"
```
Comment (TimeEngineer) [inline oxidation/client/package.json:22]: `web:copy:realease` has been deleted

File: oxidation/client/package.json [hunk 3]
```diff
@@ -3,28 +3,22 @@
...
+        "electron:copy:release": "npm run native:build -- --prod && npx cap copy @capacitor-community/electron",
         "electron:open": "npm run electron:copy && npx cap open @capacitor-community/electron",
         "electron:release": "npm run electron:copy:release && cd electron && npm run electron:release",
-        "web:libparsec:build": "cd ../bindings/web && wasm-pack build --dev --target web",
-        "web:libparsec:build:release": "cd ../bindings/web && wasm-pack build --release --target web",
-        "web:libparsec:copy": "cp -r ../bindings/web/pkg . || xcopy ..\\bindings\\web\\pkg pkg /e /s /y /i",
-        "web:copy": "npm run web:libparsec:build && npm run web:libparsec:copy",
-        "web:copy:release": "npm run web:libparsec:build:release && npm run web:libparsec:copy",
-        "web:open": "npm run web:copy && vue-cli-service serve",
+        "web:serve": "vue-cli-service serve",
```
Comment (NicoTuxx) [inline oxidation/client/package.json:21]: for consistency would be better to keep :open, like other platforms instead of serve

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: oxidation/client/package.json (hunk 1 above)
Comment (TimeEngineer) [reply oxidation/client/package.json:11]: Why do we want to hide the command behind ?

Comment (touilleMan) [reply oxidation/client/package.json:11]: yes it was a wrapper that only launched `vue-cli service build`, hence removing it to make it more clear what is actually called

File: oxidation/client/package.json (hunk 2 above)
Comment (touilleMan) [reply oxidation/client/package.json:22]: yes, it is no longer needed given the cope stuff is now done by webpack itself

Comment (TimeEngineer) [reply oxidation/client/package.json:22]: does `web:release` still work ?

File: oxidation/client/package.json (hunk 3 above)
Comment (touilleMan) [reply oxidation/client/package.json:21]: as you like, done ;-)

File: oxidation/client/package.json (hunk 2 above)
Comment (touilleMan) [reply oxidation/client/package.json:22]: yup

**Review by NicoTuxx — APPROVED** [review]

**Review by TimeEngineer — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -423,5 +423,15 @@ jobs:
...
+          # Without `VUE_CLI_TEST` set, the command hangs forever because
+          # it fails to stop the server :'(
+          # (see https://github.com/vuejs/vue-cli/issues/3401)
+          VUE_CLI_TEST=1 npm run test:e2e:headless
+          # Need to run e2e first, as it will trigger compilation of the
+          # libparsec web plugin that is also needed for the unit tests (but
+          # which doesn't trigger it compilation...)
+          npm run test:unit
         working-directory: oxidation/client
+        timeout-minutes: 10
```
Comment (FirelightFlagboy) [inline .github/workflows/ci.yml:437]: Why not splitting the this step in 2 ? one for `e2e` and one for `unit-test` ?

Comment (touilleMan) [reply .github/workflows/ci.yml:437]: done.

**Pull request merged**